Added the Port support permissions documentation (#2823)
port-labs/port-docs: docs/sso-rbac/users-and-teams/manage-users-teams.md

For example, you can define that all `Members` can create new entities from a specific blueprint, while only `Moderators` can edit them.  
For more information and examples, see the [catalog RBAC](/build-your-software-catalog/set-catalog-rbac/) section.

## Support user permissions

Port's support team may need access to your organization to provide technical assistance. You can control this access through support user permissions.

### Access duration

> Port support user access is enabled by default, you can choose to provide different access to your organization: **1 week**, **1 month**, **1 year**, **Permanent access (default)**.

Reviewer: ```suggestion
Support user access is enabled by default.  
You can define how long this access will be valid: **1 week**, **1 month**, **1 year**, **Permanent access (default)**.
```

### Support user capabilities

Support users can be created with the following restrictions:

> - **Read-only access** - Support users can be created as `read-only` users.

Reviewer: no need to say "Support users" so many times, can be removed from each bullet here
